src/os/bluestore/KernelDevice.cc

```diff
@@ -548,3 +557,22 @@ int KernelDevice::invalidate_cache(uint64_t off, uint64_t len)
   return r;
 }
 
+int KernelDevice::aio_read(uint64_t off, uint64_t len, bufferlist *pbl, IOContext *ioc, bool buffered)
```

Comment:
aio only works with O_DIRECT. if you do it on a non-O_DIRECT it 
is actually synchronous (or errors out, i forget).